src/Tooltip/desktop-view.js

```diff
@@ -18,8 +20,7 @@ class DesktopView extends Component {
   }
 
   tooltipPositionEval = () => {
-    const headerHeight = document.getElementsByClassName('header')[0]
-      .offsetHeight
+    const headerHeight = document.getElementById(HEADER).offsetHeight
     const markdownBody = document.getElementsByClassName('markdown-body')[0]
```

Comment:
Theoretically we can have more than 1 element with `markdown-body` class because it is the class that applies markdown css rules and there is no guaranties that  thepage body will always be the first one. Maybe we need to add unique id there and place it to consts file?

Reply:
@algomaster99 are you going to address this?

Reply:
@shcheklein i think it is better to address it here https://github.com/iterative/dvc.org/issues/714

Reply:
@algomaster99 It does not seem to be related to #714 . It's about changing Class to unique Id in the first place, right? and only after that about moving something to the const file.